sema/commons/j2/j2_functions.py

```
@@ -24,20 +25,28 @@ def all():
+    content: any,
+    quote: str,
+    type_name: str,
+    suffix: str | None = None,
+) -> str:
     if suffix is None:
         suffix = "^^" + type_name
     return quote + str(content) + quote + suffix
 
 
-def xsd_format_boolean(content, quote, suffix):
+def xsd_format_boolean(content: Any, quote: str, *_: Any) -> str:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `content`

(ANN401)